### PR TITLE
Forward fix unexpected success linalg.pinv.singular

### PR DESCRIPTION
**Pull request by ezyang**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #91892

Signed-off-by: Edward Z. Yang <ezyang@fb.com>

cc @mlazos @soumith @voznesenskym @yanboliang @penguinwu @anijain2305 @EikanWang @jgong5 @Guobing-Chen @chunyuan-w @XiaobingSuper @zhuhaozhe @blzheng @Xia-Weiwen @wenzhe-nrv @jiayisunx @peterbell10 @desertfire

### CODE REVIEW

**Review by anjali411 — APPROVED** [review]

**Review by janeyx99 — APPROVED** [review]
cool

**Review by Skylion007 — APPROVED** [review]

**Comment by ezyang** [comment]
narrator: it did not work

**Review by malfet — APPROVED** [review]

File: test/inductor/test_torchinductor_opinfo.py [hunk 1]
```diff
@@ -298,7 +298,6 @@ def process(device_type):
     "linalg.lstsq.grad_oriented": {f32, f64},
     "linalg.matrix_rank": {f32, f64},
     "linalg.matrix_rank.hermitian": {f32, f64},
-    "linalg.pinv.singular": {f32, f64},
```
Comment (janeyx99) [inline test/inductor/test_torchinductor_opinfo.py:301]: Looks like line 219 should be removed as well

Comment (ezyang) [reply test/inductor/test_torchinductor_opinfo.py:301]: yup agreed

**Comment by ezyang** [comment]
og got reverted, abandoning.

cc @anjali411

**Pull request closed without merging**